google/datalab/contrib/bigquery/commands/_bigquery.py

```diff
@@ -183,9 +183,10 @@ def _get_execute_parameters(load_task_id, bq_pipeline_input_config,
         if 'csv' in bq_pipeline_input_config:
           execute_task_config['csv_options'] = bq_pipeline_input_config.get('csv')
 
-    # Stuff from the transformation config
     query = utils.commands.get_notebook_item(bq_pipeline_transformation_config['query'])
-    execute_task_config['sql'] = query.sql
+    # The user's sql could span multiple lines. Here, we replace new-lines (and the carriage-return)
+    # with spaces because it could result in malformed python.
+    execute_task_config['sql'] = query.sql.replace('\n', ' ').replace('\r', ' ')
```

Comment:
Can we just wrap user's sql with triple quotes in the generated code instead?

Reply:
Yes, we could. The fix for that will require some thought though. Currently, this gets passed to the underlying object model (i.e. the - soon to be renamed - Pipeline class). There, it gets formatted as a string:
https://github.com/googledatalab/pydatalab/blob/51c668fe8e7658f64e1cfdbc76ebc66c9f6cfbfa/google/datalab/contrib/pipeline/_pipeline.py#L171

I see two choices - we could either have every string parameter be formatted with three quotes. My concern about that it could make the pipeline spec hard to read. The other choice of course is to make the Pipeline object model aware that this is a sql query string, and that it would need to be triple quoted. That breaks encapsulation somewhat. 

This fix so far feels like a better compromise.